src/tests/clipperUI/panels/signInPanel_tests.tsx

```diff
@@ -92,6 +92,54 @@ export class SignInPanelTests extends TestModule {
 
 			strictEqual(type, AuthType.OrgId, "The OrgId auth type should be recorded");
 		});
+
+		test("The 'more' button is enabled when a sign-in failure is detected (OrgID)", () => {
```

Comment:
Looks like we don't have any tab tests for this panel. Can you please add them? :)